manifests/sysmgmt.yaml

```
@@ -134,13 +134,13 @@ spec:
     namespace: services
   - name: cray-bos
     source: csm-algol60
-    version: 2.33.0
+    version: 2.34.1
```

Pull in fixes for CASMCMS-9289 and CASMCMS-9290
```suggestion
    version: 2.34.2
```